plone/app/layout/permissions.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from AccessControl.SecurityInfo import ModuleSecurityInfo
+from Products.CMFCore.permissions import setDefaultRoles
+
+
+security = ModuleSecurityInfo('plone.app.layout')
+
+security.declarePublic('ShowToolbar')
```

Comment:
Just a wild guess, but if this is the 'id' of the permission, shouldn't it be 'plone.showToolbar' like on the ZCML?